backend/index.js

```diff
@@ -0,0 +1,73 @@
+const express = require("express");
+const app = express();
+const port = 3000;
+const bodyParser = require("body-parser");
+const fs = require('fs');
+const cors = require('cors')
+
+app.use(cors())
+
+app.use(bodyParser.urlencoded({ extended: false }))
+
+app.use(bodyParser.json())
+
+app.use(bodyParser.urlencoded({ extended: true }));
+
+
+let tasks = []
+app.get("/task/getAll", (req, res) => {
+    res.json(tasks.map(x => x));
+});
+
+const saveToDataBase = []
+
+app.post("/task/createTask", (req, res) => {
+
+    let itemIds = tasks.map(x => x.id)
+    let newId = itemIds.length > 0 ? Math.max.apply(Math, itemIds) + 1 : 1;
+    const toDo = {
```

Comment:
To discuss: Maybe(or not) we should also show them what destructuring is:
`const { task, deadline, priority, category } = req.body`

What do you think ?